### DISCUSSION TITLE
Are there any programmable interfaces that can get all top-level dependencies when migrating packages.config to packagereference?

### DISCUSSION THREAD

**Discussion by Smartisa** [NuGet/Home#11656] [Ideas]
For example:
![image](https://user-images.githubusercontent.com/47079990/157184221-a5723f26-e0f1-406f-863e-3ba17f6261ca.png)
How can I automatically obtain top-level dependencies?

**Comment by nkolev92** [comment]
Try: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.PackageManagement/Utility/PackageGraphAnalysisUtilities.cs

NuGet.PackageManagement is the package.

**Reply by Smartisa** [reply]
Thanks a lot!!!
